src/AppRoutes.tsx

```diff
@@ -46,6 +47,7 @@ export const AppRoutes = () => (
     {/* entity pages */}
     <Route path="products" element={<ProductsPage />} />
     <Route path="projects" element={<ProjectsPage />} />
+    <Route path="projects/:id" element={<ProjectDetailPage />} />
```

Comment:
@jakubvanko @jomrazek WDYT about using nested routes in cases like this? https://reactrouter.com/docs/en/v6/getting-started/concepts#nested-routes